app/lib/project/MetapolatorProject.js

```diff
@@ -626,6 +627,10 @@ define([
         }
     };
 
+    _p.saveAs = function(blob, filename) {
```

Comment:
I don't think metapolatorProject is a good fit for this.  This is just "saving" data, it's not specific for a Metapolator Project at all. It may not even work within NodeJS, that might be the reason for the tests not passing at the moment.

Rather we can add it to `metapolatorStandAlone` and just export it from there for use from the UI.

If this is really not loading within NodeJS. we should even consider adding a dummy package that raises a NotImplentedError when trying to use the saveAs method or so.